src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,94 @@
-        </td>
-      </tr>
+      <tbody>
+        {todos.map(todo => {
+          const {
+            id,
+            title,
+            completed,
+          } = todo;
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+          return (
+            <tr
+              data-cy="todo"
+              className={cn({
+                'has-background-info-light': selectedTodo === id,
+              })}
+              key={id}
+            >
+              <td className="is-vcentered">{id}</td>
+              <td className="is-vcentered">
+                {completed && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+              </td>
+              <td className="is-vcentered is-expanded">
+                <p
+                  className={
+                    cn({
+                      'has-text-success': completed,
+                      'has-text-danger': !completed,
+                    })
+                  }
+                >
+                  {title}
+                </p>
+              </td>
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                  onClick={() => setSelectedTodo(id)}
+                >
+                  <span className="icon">
+                    <i
+                      className={
+                        cn(
+                          'far',
+                          {
+                            'fa-eye': !(selectedTodo === id),
+                            'fa-eye-slash': selectedTodo === id,
```

Comment:
suggest to make variable for this exprassion and use in all places
```
const isSelected = selectedTodo === id;
```
```suggestion
                            'fa-eye-slash': isSelected
```